test-variables.yaml.example

```diff
@@ -25,6 +25,10 @@ OCP_LDAP_DEV:
   AUTH_TYPE: "ldap-provider-qe"
   USERNAME: ldap_user
   PASSWORD: abc123XYZ!
+OCP_DBAAS_USER::
```

Comment:
Why did you add it to the yaml.example file?

Reply:
It should added as this would be reference template for the user. Please let me know if this needs to be addressed, but IMO this should be fine.

Reply:
OK got it